src/kaocha/specs.clj

```diff
@@ -44,10 +77,89 @@
     (s/coll-of fn? :into [])
     #(s/gen #{[]})))
 
+
+(s/def :kaocha.plugin.randomize/randomize?
+  any?)
+
+(s/def :kaocha.plugin.capture-output/capture-output?
+  any?)
+
+(s/def :kaocha/tests (s/coll-of :kaocha/testable))
+
+(s/def :kaocha/testable (s/keys :opt [:kaocha.testable/type
```

Comment:
I'm wondering if this should be decomposed further into a "test" from config and a "testable" that is the result of load-testables.

Reply:
Don't we already have that? `:kaocha/testable` vs `:kaocha.test-plan/testable`?

Reply:
oh, yes! my bad.